java/Cherepnin_Oleh/src/com/zoolatech/lecture4/tasks/_7/TaskSeven.java

```diff
@@ -0,0 +1,36 @@
+package com.zoolatech.lecture4.tasks._7;
+
+import java.util.Scanner;
+
+/**
+ * Create a method that accepts a string of digits and after each digit appends its string representation in parentheses
+ * Input: “12345”
+ * Output: “1(one)2(two)3(three)4(four)5(five)”
+ */
+public class TaskSeven {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Input numbers:");
+        System.out.println(convert(scanner.next()));
+    }
+
+    private static String convert(String string) {
+        String[] strs = string.split("");
```

Comment:
`string.split` creates a regular expressions internally, which is a heavy operation. Method `toCharArray` has a better performance